src/settings/components/settings-header.tsx

```diff
@@ -26,25 +27,31 @@ type SettingsHeaderProps = {
 	isDirty: boolean;
 };
 
-const SettingsHeader: FC<SettingsHeaderProps> = ({ onSave, onCancel, isDirty, title }) => {
+const SettingsHeader = ({ onSave, onCancel, isDirty, title }: SettingsHeaderProps): JSX.Element => {
 	const t = getT();
 	const history = useHistory();
-	const useparam = useParams();
-	const crumbs = [
-		{
-			id: 'settings',
-			label: t('settings.app', 'Settings'),
-			click: (): void => history.push(`/${SETTINGS_APP_ID}/`)
-		},
-		{
-			id: 'general',
-			label: title,
-			click: (): void => history.push(`/${SETTINGS_APP_ID}/`)
-		}
-	];
-	const search = history.location?.search;
+	const params = useParams();
+	const crumbs = useMemo(
```

Comment:
these crumbs, that are used for each module, do not work...we need to open an issue?

Reply:
Good catch, I didn't notice. Yep, we should, I'm on it.

Reply:
SHELL-81 opened for the breadcrumb not working well